testbed/gtk/flutter/generated_plugins.cmake

```diff
@@ -0,0 +1,7 @@
+list(APPEND FLUTTER_PLUGIN_LIST
+)
+
+foreach(plugin ${FLUTTER_PLUGIN_LIST})
+  add_subdirectory(flutter/ephemeral/.plugin_symlinks/${plugin}/linux plugins/${plugin})
```

Comment:
For now this will need to be `gtk`, not `linux`, since the plan is to land the parallel implementations in `gtk` and then move them.